source/documentation/getting_started/quick_start.md

```diff
@@ -1,10 +1,37 @@
 ## Quick start guide
 
-Registers in beta are reliable and are ready for use in products and services.
+Registers which are in beta are reliable and ready for use in products and services. Each register has an open, RESTful API which you can use to access the data in the register. 
```

Comment:
Should we be describing this as a "RESTful" or a "REST" API? Information I've read online and conversations I've had are conflicting.

Reply:
if it helpls users frame the type of API, I don't see why not.